ush/config_defaults.yaml

```diff
@@ -1918,7 +1928,7 @@ task_run_fcst:
   QUILTING: true
   PRINT_ESMF: false
 
-  PE_MEMBER01: '{{ OMP_NUM_THREADS_RUN_FCST * (LAYOUT_Y * LAYOUT_X + WRTCMP_write_groups * WRTCMP_write_tasks_per_group) if QUILTING else OMP_NUM_THREADS_RUN_FCST * (LAYOUT_Y * LAYOUT_X)}}'
+  PE_MEMBER01: '{{ fire.FIRE_NUM_TASKS + OMP_NUM_THREADS_RUN_FCST * (LAYOUT_Y * LAYOUT_X + WRTCMP_write_groups * WRTCMP_write_tasks_per_group) if QUILTING else OMP_NUM_THREADS_RUN_FCST * (LAYOUT_Y * LAYOUT_X) + fire.FIRE_NUM_TASKS }}'
```

Comment:
Just to keep it clearer, would you mind changing it so that the `fire.FIRE_NUM_TASKS` entry ends up in the same position in both of these equations? I personally prefer seeing it show up at the end, but consistency is preferable either way.